data/base-consumables.yaml

```diff
@@ -0,0 +1,74 @@
+Crossing:
```

Comment:
YAML files _should_ start with `---` on its own line. It won't cause an error if it's missing, but it's best practice.

Reply:
done!